server.tf

```diff
@@ -0,0 +1,303 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#------------------------------#
+# Random string for deployment #
+#------------------------------#
+resource "random_id" "random_project_id_suffix" {
```

Comment:
This name is stale; I don't see this being used for anything related to GCP projects. Old copy/paste?

Reply:
This is being used in the locals.tf, I'm going to be move it there so the context is appropiate.